### PR TITLE
docs: Maintenance

### PR DESCRIPTION
**Pull request by hifabienne**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/support/software-release-cycles-support.md [hunk 1]
```diff
@@ -70,3 +70,23 @@ During this phase, support is limited as we focus on testing and bug fixing.
 Generally available features are available to everyone and have the appropriate test coverage to be used for critical tasks.
 The software will be backwards-compatible with previous versions, for exceptions we will publish a [technical advisory](https://zitadel.com/docs/support/technical_advisory).
 Features in General Availability are not marked explicitly.
+
+## Maintenance
+
+On ZITADEL Cloud we follow a regular deployment cycle to ensure our product remains up-to-date, secure, and provides new features.
```
Comment (mffap) [inline docs/docs/support/software-release-cycles-support.md:76]:
```suggestion
ZITADEL Cloud follows a regular deployment cycle to ensure our product remains up-to-date, secure, and provides new features. 
```

File: docs/docs/support/software-release-cycles-support.md [hunk 2]
```diff
@@ -70,3 +70,23 @@ During this phase, support is limited as we focus on testing and bug fixing.
...
 The software will be backwards-compatible with previous versions, for exceptions we will publish a [technical advisory](https://zitadel.com/docs/support/technical_advisory).
 Features in General Availability are not marked explicitly.
+
+## Maintenance
+
+On ZITADEL Cloud we follow a regular deployment cycle to ensure our product remains up-to-date, secure, and provides new features. 
+Our standard deployment cycle occurs every two weeks, during which we implement updates, bug fixes, and enhancements to improve the functionality and performance of our product.
+In certain circumstances, we may require additional deployments beyond the regular two-week cycle.
+This can occur for example when we have substantial updates or feature releases that require additional time for thorough testing and implementation or security fixes.
+During these deployments, we strive to minimize any disruptions and do not expect any downtime.
```
Comment (mffap) [inline docs/docs/support/software-release-cycles-support.md:80]:
```suggestion
During deployments, we strive to minimize any disruptions and do not expect any downtime.
```

File: docs/docs/support/software-release-cycles-support.md [hunk 3]
```diff
@@ -70,3 +70,23 @@ During this phase, support is limited as we focus on testing and bug fixing.
...
+
+### Release Deployment with Risk of Downtime
+
+In rare situations where deploying a release carries a risk of downtime, we have a well-defined procedure in place to ensure transparent communication.
```
Comment (mffap) [inline docs/docs/support/software-release-cycles-support.md:84]:
```suggestion
In rare situations where deploying releases that may  carry a risk of increased latency or short downtime, we have a well-defined procedure in place to ensure transparent communication. 
```

File: docs/docs/support/software-release-cycles-support.md [hunk 4]
```diff
@@ -70,3 +70,23 @@ During this phase, support is limited as we focus on testing and bug fixing.
...
+Our standard deployment cycle occurs every two weeks, during which we implement updates, bug fixes, and enhancements to improve the functionality and performance of our product.
+In certain circumstances, we may require additional deployments beyond the regular two-week cycle.
+This can occur for example when we have substantial updates or feature releases that require additional time for thorough testing and implementation or security fixes.
+During these deployments, we strive to minimize any disruptions and do not expect any downtime.
+
+### Release Deployment with Risk of Downtime
+
+In rare situations where deploying a release carries a risk of downtime, we have a well-defined procedure in place to ensure transparent communication. 
+Prior to such deployments, we publish information on our status page, which can be accessed by visiting [https://status.zitadel.com/](https://status.zitadel.com/).
+We also recommend subscribing to those updates on the [status page](https://status.zitadel.com/).
```
Comment (mffap) [inline docs/docs/support/software-release-cycles-support.md:86]:
```suggestion
We also recommend that you subscribe to those updates on the [status page](https://status.zitadel.com/).
```

**Review by mffap — DISMISSED** [review]
some small suggestions, would also be good to go now

**Comment by mffap** [comment]
@hifabienne while we're here... do you want to include a link to the stable version in this pr? same page...

**Comment by hifabienne** [comment]
> @hifabienne while we're here... do you want to include a link to the stable version in this pr? same page...

I added a section about the release types

File: docs/docs/support/software-release-cycles-support.md [hunk 5]
```diff
@@ -34,39 +34,96 @@ If you encounter issues with an enterprise supported feature and you are eligibl
...
+Release candidates are accessible for our open source community, but will not be deployed to the ZITADEL Cloud Platform.
+
+### Latest
+
+The "latest" release channel is designed for users who prefer to access the most recent updates, features, and enhancements as soon as they become available.
+It provides early access to new functionalities and improvements but may involve a higher degree of risk as it is the most actively developed version.
+Users opting for the latest release channel should be aware that occasional bugs or issues may arise due to the ongoing development process.
+
+Latest releases will be deployed to the ZITADEL Cloud service.
+
```
Comment (mffap) [inline docs/docs/support/software-release-cycles-support.md:97]:
```suggestion
```

**Review by mffap — CHANGES_REQUESTED** [review]

**Review by mffap — APPROVED** [review]

**Pull request merged**